studybuddies/src/environments/environment.ts

```diff
@@ -3,7 +3,11 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  name: 'desarrollo',
+  urlBackend: 'http://localhost:3000/api/v1',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
  urlBackend: "http://localhost:3000/api/v1",
```